prime-angular-frontend/src/app/modules/enrolment/pages/remote-access/remote-access.component.ts

```diff
@@ -60,26 +60,27 @@ export class RemoteAccessComponent extends BaseEnrolmentProfilePage implements O
       utilService,
       formUtilsService
     );
-
-    this.enrolment = this.enrolmentService.enrolment;
   }
 
   public get sitesFormArray(): FormArray {
```

Comment:
Looks like this could be renamed to `sites`.  Doesn't look like their is a name collision anymore.